chromium_src/components/sync/base/model_type_unittest.cc

```diff
@@ -21,7 +21,7 @@ TEST(ModelTypeTest, LowPriorityUserTypes) {
 
 // This test is supposed to fail when sync types are increased/decreased
 TEST(ModelTypeTest, ModelTypeCounts) {
-  EXPECT_EQ(static_cast<int>(ModelTypeForHistograms::kMaxValue), 67);
+  EXPECT_EQ(static_cast<int>(ModelTypeForHistograms::kMaxValue), 68);
```

Comment:
this is for plus address sync which has sense for Google Account and NA for Brave